jekyll/_cci2/vs-code-extension-overview.adoc

```diff
@@ -165,12 +165,50 @@ Both of these commands can be run by:
 
 - Right-clicking anywhere within an opened `.circleci/config.yml` file.
 
-- Clicking on the CircleCI button on the top right of the editor when you have a `.circleci/config.yml`open. Note that the button will not be visible if you are editing any other file.
+- Clicking on the CircleCI button on the top right of the editor when you have a `.circleci/config.yml` open. Note that the button will not be visible if you are editing any other file.
 
 [#open-source-language-server]
 === Open source language server
 
-The config helper is based on a dedicated language server specific for CircleCI YAML files, which is open source. You can view its source code, contribute and add issues directly on the project repository: link:https://github.com/CircleCI-Public/circleci-yaml-language-server[circleci-yaml-language-server].
+The config helper is based on a dedicated language server specific for CircleCI YAML files, which is open source. You can view its source code, contribute and add issues directly on the project repository: link:https://github.com/CircleCI-Public/circleci-yaml-language-server[CircleCI YAML language server].
 
 You can also integrate the language server into any editor which supports the Language Server Protocol, and build your own plugin to benefit from config helper capabilities in your favourite editor.
 
+[#trigger-a-config-test-run]
+== Trigger a config test run
```

Comment:
If we decide to go down the "verb" route, this could become 
"Test run your config from VS Code"

And then in the following paragraph we can say:
> Note: **This feature** is currently....

Reply:
Yeah this is clearer I think so let's try it out